smpte.js

```diff
@@ -220,8 +242,54 @@ function insertNormativeReferences(docMetadata) {
   h2.innerText = "Normative references";
 }
 
+function insertTermsanddefinitions(docMetadata) {
```

Comment:
```suggestion
function insertTermsAndDefinitions(docMetadata) {
```

Reply:
"and definitions" are not capitalized on purpose per ISO guideline.

Reply:
Updated via https://github.com/SMPTE/html-pub/pull/27/commits/1d1c2d8b590b01bba6c7d31eb9773b5271602c91